packages/flutter/lib/src/widgets/spell_check.dart

```diff
@@ -99,80 +100,106 @@ class SpellCheckConfiguration {
 
 // Methods for displaying spell check results:
 
-/// Adjusts spell check results to correspond to [newText] if the only results
-/// that the handler has access to are the [results] corresponding to
-/// [resultsText].
-///
-/// Used in the case where the request for the spell check results of the
-/// [newText] is lagging in order to avoid display of incorrect results.
```

Comment:
Why did you remove this comment?  I think we need something explaining what _correctSpellCheckResults is used for.

Reply:
This was an accident! Thanks!